app/src/main/java/org/fossasia/openevent/app/core/settings/SalesDataSettings.java

```diff
@@ -0,0 +1,77 @@
+package org.fossasia.openevent.app.core.settings;
+
+import android.os.Bundle;
+import android.support.annotation.Nullable;
+import android.support.v7.preference.CheckBoxPreference;
+import android.support.v7.preference.Preference;
+import android.support.v7.preference.PreferenceManager;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+
+import com.takisoft.fix.support.v7.preference.PreferenceFragmentCompat;
+
+import org.fossasia.openevent.app.R;
+import org.fossasia.openevent.app.common.Constants;
+import org.fossasia.openevent.app.ui.ViewUtils;
+
+public class SalesDataSettings extends PreferenceFragmentCompat {
+
+    private static final String NET_SALES = "net_sales";
+    private static final String GROSS_SALES = "gross_sales";
+    private CheckBoxPreference sales;
+    private Preference.OnPreferenceChangeListener listener;
+
+    public static SalesDataSettings newInstance() {
+        return new SalesDataSettings();
+    }
+
+    @Override
+    public View onCreateView(LayoutInflater inflater, ViewGroup container, Bundle savedInstanceState) {
+        View view = super.onCreateView(inflater, container, savedInstanceState);
+        view.setBackgroundColor(getResources().getColor(R.color.color_top_surface));
+        return view;
+    }
+
+    @Override
+    @SuppressWarnings("PMD.DataflowAnomalyAnalysis") // Inevitable DU Anomaly
+    public void onCreatePreferencesFix(@Nullable Bundle bundle, String rootKey) {
+        PreferenceManager manager = getPreferenceManager();
+        manager.setSharedPreferencesName(Constants.FOSS_PREFS);
+
+        setPreferencesFromResource(R.xml.sales_data_display, rootKey);
+
+        listener = (preference, newValue) -> {
+            String key = preference.getKey();
+
+            switch (key) {
+                case GROSS_SALES:
+                    sales = (CheckBoxPreference) findPreference("net_sales");
+                    sales.setChecked(false);
+                    break;
+                case NET_SALES:
+                    sales = (CheckBoxPreference) findPreference("gross_sales");
```

Comment:
Reuse

Reply:
Sorry, I didn't get what you mean.

Reply:
`"gross_sales"` is already defined as constant